app/templates/components/event-invoice/event-info.hbs

```diff
@@ -12,9 +12,9 @@
   <div class="ui padded segment">
     <strong>{{t 'At'}} {{this.event.locationName}}</strong>
     <br>
-    <strong>{{t 'From'}}:</strong> {{header-date this.event.startsAt}}
+    <strong>{{t 'From'}}:</strong> {{general-date this.event.startsAt '' 'dddd, D MMMM, YYYY h:mm A'}}
```

Comment:
I don't remember, was this the standard format which was chosen?

Reply:
yes

Reply:
What I am found in #5747 is that at some places still time format is not standardized

Reply:
Please create a Pr to fix that as well